core/src/main/java/bisq/core/dao/state/storage/BsqBlocksStorageService.java

```diff
@@ -121,22 +128,21 @@ void copyFromResources(String postFix) {
                 log.info("Directory {} in resources does not exist.", resourceDir);
                 return;
             }
-            File dir = new File(dirUrl.toURI());
-            String[] fileNames = dir.list();
-            if (fileNames == null) {
-                log.info("No files in directory. {}", dir.getAbsolutePath());
+            FileSystem fileSystem = FileSystems.newFileSystem(dirUrl.toURI(), Collections.emptyMap());
+            List<Path> filePaths = Files.walk(fileSystem.getPath(resourceDir), 1).collect(Collectors.toList());
+            if (filePaths.size() <= 1) {
+                log.info("No files in directory. {}", dirUrl.toString());
```

Comment:
toString is not necessary.

```suggestion
                log.info("No files in directory. {}", dirUrl);
```